### PR TITLE
Portfolio: mobile version

### PR DESCRIPTION
**Pull request by GOlivierNation**
Portfolio: mobile version

This is the mobile version portfolio showing my achievement and CV to anyone who wishes to work with me.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,597 @@
+* {
```
Comment (Danieldotcomcoder) [inline assets/css/style.css:1]: One Linter check is not passing, Kindly fix it 👍 😄 

image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @GOlivierNation , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔
- Highly organized code ✔
- Nice UI Design ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: assets/css/style.css (hunk 1 above)
Comment (GOlivierNation) [reply assets/css/style.css:1]: Okay

**Review by Mre55 — APPROVED** [review]
**Hi @GOlivierNation,**

# Status: APPROVED  🎉 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**